app/javascript/components/layout/Header.js

```diff
@@ -61,28 +61,28 @@ class Header extends React.Component {
               <Nav className="mr-auto" activeKey={this.state.activeKey}>
                 {this.props.current_user?.can_see_enroller_dashboard_tab && (
                   <Nav.Link
-                    className={`${this.state.activeKey === '/patients' ? 'nav-link-active' : 'nav-link-inactive'} py-0 ml-3`}
+                    className={`${this.state.activeKey === '/patients' ? 'nav-link-active enroller-tab' : 'nav-link-inactive'} py-0 ml-3 enroller-tab`}
```

Comment:
`enroller-tab` class is added twice here, should be good to just keep it at the end so it's always there

Reply:
same with the other nav links

Reply:
[89fcedc](https://github.com/SaraAlert/SaraAlert/pull/1243/commits/89fcedce9790474e5e9e59c635858425cfb49098)